### PR TITLE
Closes #652: Add dependabot support for Github actions & Dockerfile

### LINKED ISSUE #652: Enable dependabot for Github actions workflow files
## Motivation
Similar to what was done in [this](https://github.com/az-digital/az_quickstart/issues/2157) Quickstart Issue, enabling dependabot for our workflow files would make sure we stay up to date on the current versions of community Github Actions.

### Is your feature request related to a problem? Please describe.
See above.

### Describe the solution you'd like
Add a section to the `.github/dependabot.yml` file so dependabot monitors our community Github Actions.

### PR DESCRIPTION
**Pull request by jwmoore1**
Enable dependabot alerts for Github actions. Closes #652

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -5,3 +5,10 @@ updates:
     schedule:
       interval: "monthly"
     open-pull-requests-limit: 10
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "weekly"
+    reviewers:
+      - "az-digital/developers"
```
Comment (joeparsons) [inline .github/dependabot.yml:14]: Something I wish I would have added to the Quickstart config (and think we should still add there, too):
```suggestion
      - "az-digital/developers"
    labels:
      - "ci"
```

**Review by joeparsons — APPROVED** [review]

File: .github/dependabot.yml [hunk 2]
```diff
@@ -5,3 +5,12 @@ updates:
     schedule:
       interval: "monthly"
     open-pull-requests-limit: 10
+  - package-ecosystem: "github-actions"
```
Comment (mmunro-ltrr) [inline .github/dependabot.yml:8]:
```suggestion
  - package-ecosystem: "docker"
      directory: "/"
    schedule:
      interval: "weekly"
    reviewers:
      - "az-digital/developers"
    open-pull-requests-limit: 10
    labels:
      - "ci"
  - package-ecosystem: "github-actions"
```

**Review by joeparsons — APPROVED** [review]

**Review by bberndt-uaz — DISMISSED** [review]

**Review by trackleft — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

File: .github/dependabot.yml [hunk 3]
```diff
@@ -5,3 +5,21 @@ updates:
     schedule:
       interval: "monthly"
     open-pull-requests-limit: 10
+  - package-ecosystem: "docker"
+      directory: "/"
```
Comment (trackleft) [inline .github/dependabot.yml:9]:
```suggestion
    directory: "/"
```

**Review by trackleft — APPROVED** [review]

File: .github/dependabot.yml (hunk 3 above)
Comment (jwmoore1) [reply .github/dependabot.yml]: Sorry Chris, I didn't see this till after I fixed the error :disappointed:

**Review by mmunro-ltrr — APPROVED** [review]

**Pull request merged**